apps/web/components/apps/SkeletonLoader.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { ShellSubHeading } from "@components/Shell";
+
+function SkeletonLoader() {
+  return (
+    <>
+      <ShellSubHeading title={<div className="h-6 w-32 rounded-md bg-gray-300"></div>} />
+      <ul className="animate-pulse divide-y divide-neutral-200 border border-gray-200 bg-white ">
+        <SkeletonItem />
+        <SkeletonItem />
+        <SkeletonItem />
+      </ul>
+    </>
+  );
+}
+
+export default SkeletonLoader;
+
+function SkeletonItem() {
+  return (
+    <li className="group flex w-full items-center justify-between px-2 py-4 sm:px-4">
+      <a className="flex-grow truncate text-sm" title="Google Meet " href="/event-types/9">
```

Comment:
II'm assuming this is not intentional here to have title and href like so?

Reply:
Correct :)